docs/quickstart.html

```diff
@@ -173,9 +173,14 @@ <h4 class="anchor-heading">
         </p>
 
         <p>
-            First, make sure to add <code class="language-bash">connect-file-{{fullDotVersion}}.jar</code> to the <code>plugin.path</code> property in the Connect worker's configuration (see <a href="#connectconfigs_plugin.path">plugin.path</a> for examples).
+            First, make sure to add <code class="language-bash">connect-file-{{fullDotVersion}}.jar</code> to the <code>plugin.path</code> property in the Connect worker's configuration.
+            For the purpose of this quickstart we'll use a relative path and consider the connectors' package as an uber jar.
+            However, it's worth noting that for production deployments using absolute paths is always preferable. See <a href="#connectconfigs_plugin.path">plugin.path</a> for a detailed description of how to set this config.
```

Comment:
Minor suggestion to be even more concrete and to avoid sending the reader to a different location in the docs:
```suggestion
            For the purpose of this quickstart we'll use a relative path and consider the connectors' package as an uber jar,
            which works when the quickstart commands are run from the installation directory.
            However, it's worth noting that for production deployments using absolute paths is always preferable. See <a href="#connectconfigs_plugin.path">plugin.path</a> for a detailed description of how to set this config.
        </p>

        <p>
            Edit the <code class="language-bash">config/connect-standalone.properties</code> file, add or change the <code>plugin.path</code> configuration property match the following, and save the file:
```